src/mbed_tools/cli/env_cli.py

```diff
@@ -1,42 +0,0 @@
-#
-# Copyright (C) 2020 Arm Mbed. All rights reserved.
-# SPDX-License-Identifier: Apache-2.0
-#
-"""Command to print information about environment variables used by the tools."""
-import click
-import pdoc
-from typing import Iterable
-
-from mbed_tools.targets.env import env_variables as devices_env_variables
-
-
-@click.command()
-def cli() -> None:
-    """Prints information about environment variables of all child mbed packages."""
-    click.echo(_build_output(devices_env_variables))
-
-
-_OUTPUT_PREAMBLE = """These variables used by Mbed Tools can be set either directly via environment variables or
```

Comment:
Where can I find this OUTPUT_PREAMBLE stuff in docs now?

Reply:
We can find it here - https://github.com/ARMmbed/mbed-tools/pull/75/files#diff-a650c08c2a3b6de7e502c89ee35ef1aeR129